blazar_dashboard/content/leases/forms.py

```diff
@@ -157,37 +167,34 @@ class CreateForm(forms.SelfHandlingForm):
     )
 
     def handle(self, request, data):
-        if data['resource_type'] == 'host':
-            reservations = [
+        reservations = []
+        if data['resource_type_host'] == True:
+            reservations.append(
                 {
                     'resource_type': 'physical:host',
                     'min': data['min_hosts'],
                     'max': data['max_hosts'],
                     'hypervisor_properties': '',
                     'resource_properties': '',
-                }
-            ]
-        elif data['resource_type'] == 'instance':
-            reservations = [
+                })
+        if data['network_ip_count'] > 0:
+            reservations.append(
                 {
-                    'resource_type': 'virtual:instance',
-                    'amount': data['amount'],
-                    'vcpus': data['vcpus'],
-                    'memory_mb': data['memory_mb'],
-                    'disk_gb': data['disk_gb'],
-                    'affinity': False
+                    'resource_type': 'virtual:floatingip',
+                    'network_id': 'd54938e8-24fe-486a-b267-eab048974a7c',
```

Comment:
Can we make this configurable? Or, did it work to set this to be `public` instead of a UUID (using the name instead of the UUID?)

Reply:
I can try using public to see if that works. Let me know if you think it may change enough to make it configurable and I can add a property to custom settings.